create-container-image-job.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: load-container-image
+  namespace: quay-perf
 spec:
   parallelism: 100
```

Comment:
Should reduce this to 10.

Reply:
ok, so this might be due to the image I was using, this might be an OK concurrency.